packages/space-opera/src/components/model_viewer_preview/model_viewer_preview.ts

```diff
@@ -247,21 +252,19 @@ export class ModelViewerPreview extends ConnectedLitElement {
     if (!this.modelViewer || !this.modelViewer.loaded) {
       throw new Error('onModelVisible called before mv was loaded');
     }
-    dispatchInitialCameraState(getCameraState(this.modelViewer));
+    reduxStore.dispatch(
+        dispatchInitialCameraState(getCameraState(this.modelViewer)));
   }
 
   private onCameraChange() {
-    if (!this.modelViewer) {
```

Comment:
because we're just toggling the state, didn't see a need to keep this check now

Reply:
The fact that there are so many of these errors sprinkled around the code is odd to me. Might be worth asking Steven what they're about.

Reply:
from what he's said in the past, it appears they went down this route, instead of having a boolean expression like isLoaded somewhere that handles this.